php_phongo.c

```diff
@@ -916,12 +918,7 @@ bool phongo_stream_socket_check_closed(mongoc_stream_t *stream) /* {{{ */
 	php_phongo_stream_socket *base_stream = (php_phongo_stream_socket *)stream;
 	TSRMLS_FETCH_FROM_CTX(base_stream->tsrm_ls);
 
-	return PHP_STREAM_OPTION_RETURN_OK == php_stream_set_option(base_stream->stream, PHP_STREAM_OPTION_CHECK_LIVENESS, 0, NULL);
-} /* }}} */
-
-mongoc_stream_t* phongo_stream_get_base_stream(mongoc_stream_t *stream) /* {{{ */
```

Comment:
This function isn't needed and was a fool's errand to begin with based on misunderstanding, sorry!